test/validator/samples/a11y-aria-proptypes-tokenlist/input.svelte

```diff
@@ -3,5 +3,5 @@
 <div aria-relevant />
 <div aria-relevant={true} />
 <div aria-relevant={"false"} />
-<div aria-relevant="additions removalss" />
-<div aria-relevant="additions removalss " />
\ No newline at end of file
+<div aria-relevant="additions removals_" />
+<div aria-relevant="additions removals_ " />
```

Comment:
Similarly, these two items are intentionally incorrect tokens, instead of using a typo, I've added a trailing `_` which again should satisfy the requirement of not being a valid token, while not also being a misspelled word.